data/scanners_tls.xml

```diff
@@ -215,6 +216,354 @@
 	</scanner>
 	<!-- ##### END Velodyne VLP-16 ("Puck") ##### -->
 
+	<!-- ##### BEGIN Velodyne HDL-64E ##### -->
+
+	<scanner 	id 						= "velodyne_hdl-64e"
+			accuracy_m					= "0.02"
+			beamDivergence_rad 			= "0.0007"
+			name              			= "Velodyne HDL-64E"
+			optics            			= "rotating"
+			pulseFreqs_Hz				= "20312"
+			pulseLength_ns     			= "10"
+			rangeMin_m					= "0.0100"
+			rangeMax_m					= "120"
+			scanAngleMax_deg   			= "1"
+			scanAngleEffectiveMax_deg	= "1"
+			scanFreqMin_Hz				= "0"
+			scanFreqMax_Hz				= "0"
+			wavelength_nm				= "905"
```

Comment:
also 903 nm according to https://www.goetting-agv.com/dateien/downloads/63-9194_Rev-G_HDL-64E_S3_Spec%20Sheet_Web.pdf

.. not that it makes a difference, but the correction for the VLP-16 made me wonder.